requirements-tests.txt

```diff
@@ -4,7 +4,5 @@ pytest
 coverage
 pytest-cov
 pylint
-hypothesis
-hypothesis-networkx
+pytest
```

Comment:
pytest is already in this file (sorry for not spotting it).
And also, remove attrs (see the comment)